packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx

```diff
@@ -42,12 +42,12 @@ export function MapTypeHeaderStudyDetails(props: Props) {
   const outputEntity = entities.find((entity) => entity.id === outputEntityId);
 
   const studyEntityCount = useEntityCount(
-    STUDIES_ENTITY_ID, // Study entity
+    studyEntity?.id, // Study entity
```

Comment:
Why was this better than just using the constant ID?

Reply:
The hook handles undefined, which prevents requests when the entity does not exist